recipes/clangdev/build.sh

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DBUILD_SHARED_LIBS=OFF \
+  -DLLVM_INCLUDE_TESTS=OFF \
+  -DLLVM_INCLUDE_UTILS=OFF \
+  -DLLVM_INCLUDE_DOCS=OFF \
+  -DLLVM_INCLUDE_EXAMPLES=OFF \
+  ..
+
+make -j 8
```

Comment:
I'm open to considering it, but I would need help with the logistics--if I stick a file alongisde the `meta.yaml`, will I be able to access it during the test stage? If so, how? I've added some more file tests for now, and I've checked that more commands run for now.